app/helpers/events_helper.rb

```diff
@@ -47,11 +47,11 @@ def name_of_event_type(type_id)
     type&.value || ""
   end
 
-  def train_to_teach_event_type?(type_id)
-    train_to_teach_types = [
-      GetIntoTeachingApiClient::Constants::EVENT_TYPES["Train to Teach Event"],
-      GetIntoTeachingApiClient::Constants::EVENT_TYPES["Online Event"],
-    ]
-    train_to_teach_types.include?(type_id)
+  def event_type_color(type_id)
```

Comment:
perhaps a case for a case?

Reply:
I like what you did there. Fixed up with a case statement 👍